src/components/UI/DeleteButton.tsx

```diff
@@ -16,12 +15,17 @@ const Button = styled.button`
   }
 `;
 
-function DeleteButton({ onClick, label }) {
+interface DeleteButtonProps {
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  label: string;
+}
+
+const DeleteButton: React.FC<DeleteButtonProps> = ({ onClick, label }) => {
```

Comment:
P3:
아래처럼 작성하시면 button 태그의 속성을 그대로 타입으로 이용할 수 있습니다.

```suggestion
const DeleteButton: React.FC<ButtonHTMLAttributes<HTMLButtonElement>> = ({ onClick, label }) => {
```